generated/ecs/ecs_nested.yml

```diff
@@ -14448,6 +14624,13 @@ threat:
   - threat.indicator.x509
   prefix: threat.
   reused_here:
+  - full: threat.indicator.pe
```

Comment:
I think we want these gone, right? `pe` will be nested under `file` or `process`, but not as a sibling